pkg/cmd/admin/registry/registry.go

```diff
@@ -224,6 +231,23 @@ func (opts *RegistryOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command,
 		opts.nodeSelector = valid
 	}
 
+	if len(opts.Config.FSGroup) > 0 {
+		if val, err := strconv.ParseInt(opts.Config.FSGroup, 10, 64); err != nil || val == 0 {
```

Comment:
negative values will be refused by PodSpec validation, actually having this to be set to 0 might be valid case (you want root group?).